### DISCUSSION TITLE
Things to know about finding contributors and doing a public project

### DISCUSSION THREAD

**Discussion by Ravindulakmina** [community/community#145189] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have currently created a small **Password Generator website**. It is small but I need to develop it further.
I need people's help for that. I can advise on this and , please help me by pointing out my mistakes.

You can view the product through the link below. Those who can help and point out the mistakes here, **please contribute**.
[safe-password-generator.vercel.app](https://safe-password-generator.vercel.app/)

And how can I continue this as an open-source project? Can you explain?
[Repo](https://github.com/Ravindulakmina/Password-Generator)

Try it out and give your feedback

![Screenshot 2024-11-21 at 15 26 55](https://github.com/user-attachments/assets/2529cb7f-d902-4393-bea8-26ca6fe02630)

**Comment by shinybrightstar** [comment]
Hi @Ravindulakmina, we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
